src/Compilers/CSharp/Portable/Binder/Binder.QueryUnboundLambdaState.cs

```diff
@@ -50,7 +50,7 @@ public override bool HasExplicitReturnType(out RefKind refKind, out TypeWithAnno
             public override RefKind RefKind(int index) { return Microsoft.CodeAnalysis.RefKind.None; }
             public override ScopedKind DeclaredScope(int index) => ScopedKind.None;
             public override MessageID MessageID { get { return MessageID.IDS_FeatureQueryExpression; } } // TODO: what is the correct ID here?
-            public override Location ParameterLocation(int index) { return _parameters[index].Locations[0]; }
+            public override Location ParameterLocation(int index) { return _parameters[index].TryGetFirstLocation(); }
```

Comment:
>ParameterLocation [](http://example.com/codeflow?start=37&length=17)

It looks like this API is used for error reporting, let's return Location.None instead of null.

Reply:
Just to check, do you want that in this PR, or in the "Let's open a follow-up issue to deal with the fact that location can be null here." PR?

Reply:
> Just to check, do you want that in this PR, or in the "Let's open a follow-up issue to deal with the fact that location can be null here." PR?

No preference